build/yaml/botbuilder-dotnet-sign.yml

```diff
@@ -0,0 +1,55 @@
+#
+# Replaces the classic BotBuilder-DotNet-master-Signed-daily.
+#
+
+# "name" here defines the build number format. Build number is accessed via $(Build.BuildNumber)
+name: $(Date:yyyyMMdd).$(Build.BuildId)
+
+pool:
+  vmImage: 'windows-latest' # or 'windows-2019'
+  demands:
+  - msbuild
+  - visualstudio
+
+trigger: none # ci trigger is set in ADO
+pr: none # pr trigger is set in ADO
+
+variables:
+  BuildConfiguration: Release-Windows
+  TestConfiguration: Release
+  BuildPlatform: any cpu
+  MSBuildArguments: -p:PublishRepositoryUrl=true -p:GeneratePackages=true -p:IncludeSymbols=true -p:SymbolPackageFormat=snupkg
+  Parameters.solution: Microsoft.Bot.Builder.sln
+#  PreviewPackageVersion: 5.0.0-preview-$(Build.BuildNumber) # Consumed by projects in Microsoft.Bot.Builder.sln. Define this in Azure to be settable at queue time.
```

Comment:
Do we need to uncomment this lines to apply the right preview version? or that's done somewhere else?

Reply:
This is done somewhere else. We should remove these lines, however.

Reply:
I have reworded these comments for better clarity. (No, we don't uncomment them.)
These comments are important so pipeline maintainers will know what variables must be defined externally, and how they are consumed outside the pipeline. These comments are a standard element in all of our pipelines. Viz: https://github.com/microsoft/BotBuilder-Samples/blob/1417f920913824f61f694d25e18a325e07543e3a/build/yaml/sample-dotnet-samplebot-linux-test.yml#L6